src/index.js

```diff
@@ -18,22 +19,21 @@ const sendData = async (url = '', data = {}) => {
 const nameInput = document.getElementById('name');
 const scoreInput = document.getElementById('score');
 const submitButton = document.getElementById('submit');
-
-submitButton.addEventListener('click', async () => {
+submitButton.addEventListener('submit', async () => {
   await sendData('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3qxAA7wy5DPzg65q4nWs/scores/', { user: nameInput.value, score: +scoreInput.value });
   nameInput.value = '';
   scoreInput.value = '';
 });
 
 const refreshButton = document.getElementById('refresh');
-const scoresUl = document.getElementsByClassName('boardlist')[0];
+const scoresUl = document.getElementsByClassName('boardList')[0];
 
 const loadScores = async () => {
   scoresUl.innerHTML = '';
   const { result: scores } = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3qxAA7wy5DPzg65q4nWs/scores/').then((response) => response.json());
   scores.forEach((score) => {
     const li = document.createElement('li');
-    li.innerHTML = `${score.user}: ${score.score}`;
+    li.innerHTML = `${score.user} : ${score.score}`;
     scoresUl.appendChild(li);
```

Comment:
- [ ] I am not able to send and receive data. Please make sure that submit button is working well and hitting the API.

Reply:
Changes done :)
Thanks for your review